helm_deploy/values-prod.yaml

```diff
@@ -0,0 +1,43 @@
+---
+# Per environment values which override defaults in create-and-vary-a-licence/values.yaml
+
+generic-service:
+  replicaCount: 4
+
+  ingress:
+    host: create-and-vary-a-licence.hmpps.service.justice.gov.uk
+    contextColour: green
+
+  env:
+    INGRESS_URL: "https://create-and-vary-a-licence.hmpps.service.justice.gov.uk"
+    HMPPS_AUTH_URL: "https://sign-in.hmpps.service.justice.gov.uk/auth"
+    TOKEN_VERIFICATION_API_URL: "https://token-verification-api.prison.service.justice.gov.uk"
+    COMMUNITY_API_URL: "https://community-api.probation.service.justice.gov.uk"
+    PRISON_API_URL: "https://api.prison.service.justice.gov.uk"
+    LICENCE_API_URL: "https://create-and-vary-a-licence-api.hmpps.service.justice.gov.uk"
+    PRISONER_SEARCH_API_URL: "https://prisoner-offender-search.prison.service.justice.gov.uk"
+    PROBATION_SEARCH_API_URL: "https://probation-offender-search.hmpps.service.justice.gov.uk"
+    PRISON_REGISTER_API_URL: "https://prison-register.hmpps.service.justice.gov.uk"
+    EXIT_SURVEY_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
+    PHASE_BANNER_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
+    LICENCE_WATERMARK: "false"
+    # A reference to the Gotenberg service - to submit HTML to PDF conversions (internal within namespace)
+    GOTENBERG_API_URL: "http://create-and-vary-a-licence-gotenberg"
+    # A reference to the CVL service - for links in HTML to get images/stylesheets (internal within namespace)
+    LICENCES_URL: "http://create-and-vary-a-licence"
```

Comment:
You need the ENV variable to restrict rollout areas. Although if we want to test on the prod environment then leave it for now!

Reply:
Yes, left this out for now whilst we check it. Only one environment variable required `RESTRICT_ROLLOUT=true` and would then read the probation areas and prisons from `config.js`.